### PR TITLE
English bundle graphite description

### PR DESCRIPTION
**Pull request by Arkanic**
graphite now has a description
its not much
Yay. my fist commit

### CODE REVIEW

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -570,6 +570,7 @@ item.lead.description = A basic starter material. Used extensively in electronic
 item.coal.name = Coal
 item.coal.description = A common and readily available fuel.
 item.graphite.name = Graphite
+item.graphite.description = Mineralized carbon, used as ammunition as well as electrical insulation.
```
Comment (Anuken) [inline core/assets/bundles/bundle.properties]: Should be "used *for* ammunition"; don't use "as" three times in a sentence.

Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: oof
ill fix it

Comment (Arkanic) [reply core/assets/bundles/bundle.properties]: @anuken i might start doing this more often
i hope ill get better eventually

**Comment by Anuken** [comment]
:+1:

**Pull request merged**